src/Features/CSharp/Portable/EncapsulateField/CSharpEncapsulateFieldService.cs

```diff
@@ -37,9 +37,15 @@ internal class CSharpEncapsulateFieldService : AbstractEncapsulateFieldService
             }
 
             var tempAnnotation = new SyntaxAnnotation();
-            var newIdentifier = SyntaxFactory.Identifier(originalFieldName)
-                                             .WithTrailingTrivia(declarator.Identifier.TrailingTrivia)
-                                             .WithLeadingTrivia(declarator.Identifier.LeadingTrivia);
+            var escapedName = originalFieldName.EscapeIdentifier();
+            var newIdentifier = SyntaxFactory.Identifier(
```

Comment:
The only overload of `.Identifier()` with separate `text` and `valueText` parameters requires that the leading and trailing trivia be explicitly specified.